interface/billing/new_payment.php

```diff
@@ -524,6 +539,14 @@ function FillUnappliedAmount()
     $('select').removeClass('class1 text')
 });
 </script>
+<script>
+<?php
+    // jQuery script to change state dynamically
+    $user_settings_php_path = '../../library/ajax/user_settings.php';
```

Comment:
There's got to be an automatic way to deal with this.

Reply:
Thinking ;)

Reply:
Have you tried :
`$user_settings_php_path = $GLOBALS['webroot'] ?>."/library/ajax/user_settings.php";`

Reply:
did not work

Reply:
See here for an example that calls the same script:
https://github.com/openemr/openemr/blob/master/interface/main/left_nav.php#L1890